report_qr/tests/test_report_qr.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import HttpCase
+
+
+class TestReportQr(HttpCase):
+    def test_qr_generation(self):
+        data = "TEST"
+        image = self.url_open("/report/qr?value=%s" % data)
+        self.assertEqual(image.headers["Content-type"], "image/png")
```

Comment:
maybe also test for correct encoding by decoding  the image back to text?
self.assertEqual( qrcode.Decoder().decode(image).result , "TEST")